groups/artifactory/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "account_name" {
+  description = "The name of the AWS account we're using"
+  type        = string
+}
+
+variable "environment" {
+  description = "The environment name to be used when creating AWS resources"
+  type        = string
+}
+
+variable "region" {
+  description = "The AWS region in which resources will be created"
+  type        = string
+}
```

Comment:
We should either define a suitable default value, or ensure the desired value is defined in the appropriate `vars` file.